scripts/userstyles.yml

```
@@ -211,6 +212,12 @@ userstyles:
     icon: deepl
     color: blue
     current-maintainers: [*watatomo]
+  discuit.org:
```

```suggestion
  discuit:
```

I think we can skip the tld/domain here and just use `discuit`.